README.md

```diff
@@ -45,3 +46,4 @@ Currently, two official plugins are available:
 - Michael Kleinschmidt ([@miklesch](https://github.com/miklesch))
 - Kalina Iwaszko ( [@messkalina] (https://github.com/messkalina))
 - Bhagya Samarathunga ( [@BhagyaPrasadSamarathunga] (https://github.com/BhagyaPrasadSamarathunga))
+- Iman Bajalan ([@BajalanIman](https://github.com/BajalanIman))
```

Comment:
It seems like your feature branch contains changes from the previous ticket. It's not a big deal but it's a good practice to only include changes in your PR which are directly relevant to the ticket. This makes it easier for your teammates to review your code, otherwise they might get confused, or there is just more code that needs to be reviewed.

Reply:
When I tried to push my changes, I noticed that the previous conflicts from last night were still unresolved. So, I resolved the conflicts first and then pushed all the changes. I'll definitely keep this in mind for next time.